src/components/NotificationBox.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { CSSProperties } from 'react';
+import { suomifiTheme, Icon } from 'suomifi-ui-components';
+import { Text } from 'components/ResponsiveComponents';
+import { NamespacesConsumer } from 'react-i18next';
+
+interface Props {
+  style?: CSSProperties;
```

Comment:
Tried to change text color for notification, but it seems not possible at the moment.

Reply:
Didn't really pay attention to that at this point. In my opinion that's something to be implemented once we make the actual reusable notification component.

Reply:
Good find nevertheless 👍